client/src/components/search/searchBar/sortBy/SortBy.jsx

```diff
@@ -13,17 +13,37 @@ function SortBy({SortByState, SortState }) {
     setSortBy("Lowest");
     SwitchSortOption(false);
   };
-  
+
   return (
     <div className="sort-by-holder global-box-shadow">
       <ul className="sort-by-list">
         <li>
-          <div onClick={onClickNearest} className="sort-by-items">
+          <div
+            onClick={onClickNearest}
+            style={
+              SortBy === "Nearest" ? { color: "white" } : { color: "black" }
+            }
+            className={
+              SortBy === "Nearest"
+                ? "sort-by-items sort-selected"
+                : "sort-by-items"
+            }
+          >
             Sort by Nearest
           </div>
         </li>
         <li>
-          <div onClick={onClickLowest} className="sort-by-items">
+          <div
+            onClick={onClickLowest}
+            style={
+              SortBy === "Lowest" ? { color: "white" } : { color: "black" }
+            }
+            className={
+              SortBy === "Lowest"
+                ? "sort-by-items sort-selected"
+                : "sort-by-items"
+            }
+          >
```

Comment:
Maybe color style can go in sort-selected

Reply:
I won't be able to check for variable changes then.